src/test/java/com/puppycrawl/tools/checkstyle/AstTreeStringPrinterTest.java

```diff
@@ -70,6 +71,17 @@ public void testParseFile() throws Exception {
     }
 
     @Test
+    public void testPrintAst() throws Exception {
```

Comment:
@rnveach `testParseFile` calls `AstTreeStringPrinter.printFileAst`. But this test calls `AstTreeStringPrinter.printAst`. Without `File` :) Both methods are quite similar, but `printAst`was never called from tests.